src/devices/Arduino/Arduino.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <RootNamespace>Iot.Device.Arduino</RootNamespace>
+    <Configurations>Windows-Debug;Linux-Debug;Linux-Release;Windows-Release;Debug;Release</Configurations>
```

Comment:
For now lets keep consistency and don't define these Configurations or the Conditions on the Project references below. All of it is going to go away anyway very soon when we merge your other PR, and I plan on making a tool that will fix all projects so I want to have them as consistent as possible.

Reply:
I agree. When we merge the other PR this is obsolete anyway. I'll remove this when the PR is ready to be merged (or clean it up if we merge the other one first).